app/assets/javascripts/common/password_validation.js

```diff
@@ -0,0 +1,53 @@
+PasswordValidation = function() {
```

Comment:
This is still a work in progress. The next ticket will require significant changes to this script. The console logging is intentional because it makes debugging and testing easier. It will be removed in the next ticket.